ush/config.sh.RRFS_NA_3km

```diff
@@ -175,6 +179,9 @@ COMINgfs=""
 STMP="/lfs4/BMC/nrtrr/NCO_dirs/stmp"  # Path to directory STMP that mostly contains input files.
 PTMP="/lfs4/BMC/nrtrr/NCO_dirs/ptmp"  # Path to directory STMP that mostly contains input files.
 NWGES="/lfs4/BMC/nrtrr/NCO_dirs/nwges"  # Path to directory NWGES that save boundary, cold initial, restart files
+STMP="/mnt/lfs4/BMC/wrfruc/Chunhua.Zhou/tmp/NCO_dirs/stmp"  # Path to directory STMP that mostly contains input files.
+PTMP="/mnt/lfs4/BMC/wrfruc/Chunhua.Zhou/tmp/NCO_dirs/ptmp"  # Path to directory STMP that mostly contains input files.
+NWGES="/mnt/lfs4/BMC/wrfruc/Chunhua.Zhou/tmp/NCO_dirs/nwges"  # Path to directory NWGES that save boundary, cold initial, restart files
```

Comment:
Similarly, the 2nd STMP, PTMP, NWGES for temporary testing?

Reply:
Yes, forgot to remove them before committing. Thanks for pointing out!